charts/topolvm/templates/node/daemonset.yaml

```diff
@@ -220,22 +251,23 @@ spec:
             path: {{ .Values.node.kubeletWorkDirectory }}/pods/
             type: DirectoryOrCreate
         {{- if .Values.node.lvmdEmbedded }}
-          {{ $global := . }}
-          {{- $lvmds := concat ( list .Values.lvmd ) .Values.lvmd.additionalConfigs }}
-          {{- range $lvmdidx, $lvmd := $lvmds }}
-            {{/* Undo the "." rewritten by the range block */}}
-            {{- with $global }}
         - name: config
           configMap:
-            name: {{ template "topolvm.fullname" . }}-lvmd-{{ $lvmdidx }}
-            {{- end }}
-          {{- end }}
+            name: {{ template "topolvm.fullname" . }}-lvmd
+        {{- end }}
+        {{- if .Values.node.lvmdEmbedded }}
+        - name: lvmd-socket-dir
+          emptyDir: 
+            medium: Memory
```

Comment:
nice change, but are you sure this is the only thing requiring hostPID? Just because it starts doesnt mean all block devices are recognized properly

Reply:
I did test in my environment which at the moment is EKS with k8s 1.28

Reply:
See https://github.com/topolvm/topolvm/pull/816#issuecomment-1892928961